spec/grape/jsonapi/parameters/filter_spec.rb

```diff
@@ -134,6 +134,7 @@
     end
 
     let(:model) { double(:model) }
+    let(:query) { double(:query) }
```

Comment:
should not need both query and model

because they auto exchange for one another, and having both mocked will make testing more complicated